metrics/mean_iou/mean_iou.py

```diff
@@ -290,6 +290,7 @@ def _info(self):
             reference_urls=[
                 "https://github.com/open-mmlab/mmsegmentation/blob/71c201b1813267d78764f306a297ca717827c4bf/mmseg/core/evaluation/metrics.py"
             ],
+            format="numpy",
```

Comment:
Apparently we also need to update the features:

```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-13-8df7e6987cf7> in <module>
      3 import evaluate
      4 
----> 5 metric = evaluate.load("/content/evaluate/metrics/mean_iou", keep_in_memory=True)
      6 
      7 def compute_metrics(eval_pred):

4 frames
/usr/local/lib/python3.7/dist-packages/evaluate/info.py
 in __post_init__(self)
     66                 if not isinstance(value, Value):
     67                     raise ValueError(
---> 68                         f"When using 'numpy' format, all features should be a `datasets.Value` feature. "
     69                         f"Here {key} is an instance of {value.__class__.__name__}"
     70                     )

ValueError: When using 'numpy' format, all features should be a `datasets.Value` feature. Here predictions is an instance of Sequence
```

Reply:
I think this ValueError doesn't make sense and should be removed